crates/evm/src/tests/utils.rs

```diff
@@ -323,8 +323,8 @@ pub(crate) fn get_evm_test_config() -> EvmConfig {
     config
 }
 
-pub(crate) fn get_fork_fn_only_fork1() -> impl Fn(u64) -> Fork {
-    |_: u64| Fork::new(SovSpecId::Kumquat, 0)
+pub(crate) fn get_fork_fn_only_fork2() -> impl Fn(u64) -> Fork {
```

Comment:
This has to change every time we create a new fork?

How about we implement:

````
impl SpecId {
   pub fn latest() -> Self {
        Self::Fork2
   }
}
```

And use this `SpecId::latest()` everywhere?

Reply:
https://github.com/chainwayxyz/citrea/issues/1869